### PR TITLE
[3.0] Fix column attributes removed when generate script.

### PR DESCRIPTION
**Pull request by as247**
Fix column attributes: https://github.com/yajra/laravel-datatables/issues/1380

### CODE REVIEW

File: src/Html/Builder.php [hunk 1]
```diff
@@ -116,9 +116,9 @@ public function generateScripts()
         $args = array_merge(
             $this->attributes, [
                 'ajax'    => $this->ajax,
-                'columns' => $this->collection->map(function ($column) {
-                    unset($column->attributes);
-
+                'columns' => $this->collection->map(function (Column $column) {
+                    $column=$column->toArray();
```
Comment (yajra) [inline src/Html/Builder.php]: Please add a space around `=`.

File: src/Html/Builder.php [hunk 2]
```diff
@@ -116,9 +116,9 @@ public function generateScripts()
...
+                    unset($column['attributes']);
                     return $column;
```
Comment (yajra) [inline src/Html/Builder.php]: Please add new line before return.

**Comment by yajra** [comment]
Just need to fix cs and this is good to merge. Thanks a lot!

File: src/Html/Builder.php (hunk 1 above)
Comment (yajra) [reply src/Html/Builder.php]: Also, do we really need to convert it to array? Just realized that this might be the same as before.

Comment (as247) [reply src/Html/Builder.php]: object is passed by reference, convert it to array and the link will be removed, the we are fine to unset attributes without affect to original object.

**Comment by yajra** [comment]
Still having the same issue when generating the scripts first before the table.

```php
{{$dataTable->scripts()}}
{{$dataTable->table(['class' => 'city-table table table-bordered'])}}
```

**Comment by yajra** [comment]
My bad, copy paste error. Missing semi-colon. :)

**Comment by yajra** [comment]
Released on v3.0.3, thanks a lot 🚀

**Pull request merged**